Tasks/HelmDeployV0/src/kubernetescli.ts

```diff
@@ -29,9 +29,11 @@ export default class kubernetescli extends basecommand {
     public setKubeConfigEnvVariable() {
         if (this.kubeconfigPath && fs.existsSync(this.kubeconfigPath)) {
             tl.setVariable("KUBECONFIG", this.kubeconfigPath);
+            tl.setVariable('helmExitCode', tl.TaskResult.Succeeded.toString());
```

Comment:
Use a different enum. or just have global variables for `0` and `2` right now. 
This doesn't explicitly imply that it's going to result in 0 or 2

Reply:
We are using tl.TaskResult.Succeeded in other parts of the task as well. I thought we should keep the codes used for Success and Failure consistent, that's why I'm using the same thing here as well.